README.md

```diff
@@ -1,17 +1,23 @@
 # Title <!-- Replace 'Title' and this comment with a title or name of your algorithm -->
-Description <!-- Replace this entire line with a short description of your algorithm -->
+The Greenness Transformer provides a series of calculations for rgb image data used in monitoring plant growth and health.
+A further explanation for the transformer can be found at [this link](https://docs.google.com/document/d/1cAm5w1Bs6dB1SHgf-HVmwwbebLhbZMUvTkXLtas_-xI/edit)
```

Comment:
This needs to be converted to a link for each algorithm implemented. Check with David on how to quote and reference the sources.